modules/repofiles/update.go

```diff
@@ -99,6 +99,10 @@ func detectEncodingAndBOM(entry *git.TreeEntry, repo *models.Repository) (string
 	}
 
 	result, n, err := transform.String(charsetEncoding.NewDecoder(), string(buf))
+	if err != nil {
+		// return default
```

Comment:
Maybe we should add a log here.

Reply:
There are a lot other cases where the default is returned if there was an error, should I log it there too?